pallets/funding/src/lib.rs

```diff
@@ -637,44 +623,27 @@ pub mod pallet {
 		Contribution {
 			project_id: ProjectId,
 			contributor: AccountIdOf<T>,
+			id: u32,
 			amount: BalanceOf<T>,
 			multiplier: MultiplierOf<T>,
 		},
-		/// A project is now in its community funding round
-		CommunityFundingStarted {
-			project_id: ProjectId,
-		},
-		/// A project is now in the remainder funding round
-		RemainderFundingStarted {
-			project_id: ProjectId,
-		},
-		/// A project has now finished funding
-		FundingEnded {
-			project_id: ProjectId,
-			outcome: FundingOutcome,
-		},
-		/// Something was not properly initialized. Most likely due to dev error manually calling do_* functions or updating storage
-		TransitionError {
-			project_id: ProjectId,
-			error: DispatchError,
-		},
 		ProjectOutcomeDecided {
 			project_id: ProjectId,
 			decision: FundingOutcomeDecision,
 		},
+		BidRefunded {
+			project_id: ProjectId,
+			account: AccountIdOf<T>,
+			bid_id: u32,
+			reason: RejectionReason,
+		},
```

Comment:
maybe specify the amount? since it will be deleted from storage and we would have to do another event query to find out how much was refunded